### PR TITLE
v0.81.0 -  listing states

### PR DESCRIPTION
**Pull request by sonnyprince**
_Update listing states._

screen shot 2018-09-10 at 13 44 26

- [x] This PR has been checked with regard to our brand guidelines
- [x] UI Documentation has been [created|updated]
- [x] This code has been checked with regard to our accessibility standards

## Browsers Tested

- [x] Chrome
- [x] Edge
- [x] Internet Explorer 11
- [x] Mobile

### CODE REVIEW

File: src/scss/components/_listings.scss [hunk 1]
```diff
@@ -40,9 +41,19 @@ $listing-promoIconColor         : $orange;
     }
 }
 
+.c-listing--preorder,
```
Comment (ashleynolan) [inline src/scss/components/_listings.scss]: Seems weird to be naming things with the function specifically. For example, say we wanted to re-use these styles for a listing component that wasn't for pre-order's or offline restaurants.

Would be better to use modifier that explain the styling they add `c-listing--spaced` (imagine there's a better name out there 😉 )

File: src/scss/components/_listings.scss [hunk 2]
```diff
@@ -40,9 +41,19 @@ $listing-promoIconColor         : $orange;
...
 .c-listing--offline {
-    padding: spacing(x2);
-    background: $listing-offlineBackground;
+    margin-top: spacing(x3);
+}
+
+.c-listing--preorder {
```
Comment (ashleynolan) [inline src/scss/components/_listings.scss]: Same here – should be named so it's more re-usable

File: src/scss/components/_listings.scss [hunk 3]
```diff
@@ -53,11 +64,11 @@ $listing-promoIconColor         : $orange;
         background: $listing-background;
         border-bottom: 1px solid $listing-borderColor;
 
-        .c-listing &:last-child {
-            border: 0;
+        .c-listing--offline & {
```
Comment (ashleynolan) [inline src/scss/components/_listings.scss]: Same here

File: src/scss/components/_listings.scss [hunk 4]
```diff
@@ -13,7 +13,8 @@ $listing-borderColor            : $color-border;
 $listing-iconFill               : $grey--mid;
 $listing-labelBackground        : $grey--offWhite;
 $listing-labelBorder            : 2px;
-$listing-offlineBackground      : $grey--lightest;
+$listing-preorderBackground     : $grey--lightest;
+$lighting-offlineBackground     : rgba($listing-background, 0.5);
```
Comment (DamianMullins) [inline src/scss/components/_listings.scss]: * listing

File: src/scss/components/_listings.scss [hunk 5]
```diff
@@ -40,9 +41,19 @@ $listing-promoIconColor         : $orange;
...
+    margin-top: spacing(x3);
+}
+
+.c-listing--preorder {
+    overflow: hidden;
+    padding: 0 spacing(x2) spacing(x2);
+    background: $listing-preorderBackground;
+
+    &:first-child {
+        background: red !important;;
```
Comment (DamianMullins) [inline src/scss/components/_listings.scss]: Should reference a variable for colours?

Comment (sonnyprince) [reply src/scss/components/_listings.scss]: this was me doing a stupid 🤦‍♂️  _deletes_

**Comment by sonnyprince** [comment]
@ashleynolan @DamianMullins could i get some ✅'s please :)

**Pull request merged**